stuff/pipelines/producer_contest.py

```diff
@@ -0,0 +1,99 @@
+#!/usr/bin/env python3
+
+'''Send messages to Contest queue in rabbitmq'''
+
+import argparse
+import datetime
+import logging
+import os
+import sys
+import json
+
+import mysql.connector
+import mysql.connector.cursor
+import pika
+
+from database.contest import get_contests
```

Comment:
```suggestion
import database.contest
```

[Hay que evitar el uso de `from X import Y`](https://github.com/omegaup/omegaup/wiki/Coding-guidelines#:~:text=Hay%20que%20evitar%20el%20uso%20de%20from%20module%20import%20function). se vale usar `from database import contest` y usar `contest.get_contests`.

Reply:
Tengo un detalle con este cambio. Si lo aplico, me comienza a mostrar el siguiente error:
```
E           AttributeError: <module 'producer_contest' from '/opt/omegaup/stuff/pipelines/producer_contest.py'> does not have the attribute 'get_contests'
```
porque lo estoy mockeando en la prueba:
```py3
mocker.patch('producer_contest.get_contests', return_value=params)
```

Lo intenté utilizar de las dos formas, pero obtengo el mismo resultado. Quizás exista otra forma de hacer el mock. Estaré investigando como arreglarlo.

Reply:
Listo. Agregué una función intermedia para poder obtener información de la base de datos. De esta forma ya se puede mockear en las pruebas